forest/jasmine/tests/test_traj2stats.py

```diff
@@ -399,6 +399,90 @@ def test_gps_summaries_log_format(
     assert np.all(dates_stats == dates_log)
 
 
+def test_gps_summaries_summary_vals(
+    coords1, sample_trajectory, sample_nearby_locations, mocker
+):
+    """Testing gps summaries summary values
+    are correct
+    """
```

Comment:
For one liner docstrings, please keep the closing """ on the same line
https://peps.python.org/pep-0008/#documentation-strings